src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java

```diff
@@ -54,7 +54,9 @@
 
     private static final String NONEXISTENT_CHANNEL_ID = "NONEXISTENT_CHANNEL_ID";
     private static final String NONEXISTENT_UPLOAD_ID = "NONEXISTENT_UPLOAD_ID";
+    private static final String NONEXISTENT_USER_NAME = "NONEXISTENT_USER_NAME";
     private static final String CHANNEL_ID = "CHANNEL_ID";
+    private static final String USER_NAME = "USER_NAME";
```

Comment:
super nit: USER_NAME "username" is actually a single word not 2: https://www.merriam-webster.com/dictionary/username

nit: also below MOCK_DATE isn't actually a MOCK_DATE because it's a real date not a mock or a fake.  Maybe `TEST_DATE` or just `DATE` should be sufficient